lib/organisation_import/import_organisation_data.rb

```diff
@@ -4,13 +4,6 @@
 require "open-uri"
 
 class ImportOrganisationData
-# NB: To me it doesn't seem too unreasonable to have the transformation logic here, but in the interest of not
-#     deleting comments I didn't write, see the TODO below.
-# TODO: Refactor the transformation logic into the model.
-# These are the attributes that require additional transformation before being added to the model. The first value of
-# the array is the row key name, the second is the method used for the transformation.  URL is the exception, as it
-# requires an external function call-this is handled in the method.
```

Comment:
I moved this comment to be next to the code it describes. Look [here](https://github.com/DFE-Digital/teaching-vacancies/blob/9256e4fbc25c85d3bba10618fc2c54551fb14538/lib/update_school_data.rb) to see the original context.